README.md

```diff
@@ -95,15 +95,18 @@ import mrack
 
 ## Contribute
 
-Projects is using [black](https://github.com/psf/black) formater to keep consistent
+Projects is using [black](https://github.com/psf/black) formater and [isort](https://github.com/PyCQA/isort) to keep consistent
 formatting, [flake8](https://flake8.pycqa.org/en/latest/) and
 [pydocstyle](http://pycodestyle.pycqa.org/en/latest/intro.html) to ensure following
 Python good practices.
 
-Contributions (Pull Requests) are welcome. It is expected that they will pass tox tests
-and include unit tests for new code.
+Contributions (Pull Requests) are welcome. It is expected that they will pass tox tests and code checkers.
+Inclusion of the unit tests for the new code is recommended.
+Because of that we have configured [pre-commit](https://github.com/pre-commit/pre-commit) hook.
```

Comment:
Isn't it better to point to https://pre-commit.com/ , i.e. one less click?

Reply:
Let me change it to the official site.